### DISCUSSION TITLE
Failed To Collect Paga Data for /api/webhook/clerk

### DISCUSSION THREAD

**Discussion by mishra-gaurav-04** [prisma/prisma#22460] [Q&A]
### Summary
I have a next js application it's api routes are working in dev mode but on deploying it on vercel it is giving me an error of Failed to collect page data for /api/webhook/clerk ,I am using prisma ORM for handing the crud operation and mongo db is being used as database. Any suggestion regarding this 
![Screenshot from 2023-12-19 00-33-36](https://github.com/prisma/prisma/assets/108576654/13bacc5a-80f5-4674-b0cb-1ae8c1b0eef3)

Any suggestion regarding this will be very helpful 
Thanks in advance

### **Prisma Schema **
```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

model User {
  id        String   @id @default(auto()) @map("_id") @db.ObjectId
  clerkId   String   @unique
  email     String   @unique
  userName  String   @unique
  firstName String
  lastName  String
  photo     String
  events    Events[]
  orders    Orders[]
}

model Events {
  id            String   @id @default(auto()) @map("_id") @db.ObjectId
  title         String
  description   String
  location      String
  createdAt     DateTime @default(now())
  imageUrl      String
  startDateTime DateTime @default(now())
  endDateTime   DateTime @default(now())
  price         String
  isFree        Boolean  @default(false)
  url           String
  organizerId   String   @db.ObjectId
  organizer     User     @relation(fields: [organizerId], references: [id])
  orders        Orders[]
}

model Category {
  id   String @id @default(auto()) @map("_id") @db.ObjectId
  name String @unique
}

model Orders {
  id          String   @id @default(auto()) @map("_id") @db.ObjectId
  createdAt   DateTime @default(now())
  stripeId    String   @unique
  totalAmount String
  buyer       User     @relation(fields: [buyerId], references: [id])
  buyerId     String   @db.ObjectId

  event   Events @relation(fields: [eventId], references: [id])
  eventId String @db.ObjectId
}
```

**Comment by ludralph** [comment]
Hi @mishra-gaurav-04 👋 

Thank you for raising this question.

Can you please provide the complete error trace to help debug this further? Also, this looks like a Next.js specific error, could you check this [discussion](https://github.com/vercel/next.js/discussions/50955) to see if any of the solutions there work for you?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
